sdk/identity/Azure.Identity/src/AzureIdentityEventSource.cs

```diff
@@ -407,6 +407,7 @@ public void ServiceFabricManagedIdentityRuntimeConfigurationNotSupported()
             }
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode", Justification = "Parameters to this method are primitive and are trimmer safe.")]
```

Comment:
This was causing AOT failures for me.   Since the parameters are all primitives, it should be safe to annotate, following the pattern for `AuthenticatedAccountDetailsEvent`.

@m-redding:  Please double-check me.

Reply:
Looks good!!